src/main/java/parking/helper/PasswordSimulator.java

```diff
@@ -5,7 +5,7 @@
 import java.util.Random;
 
 public class PasswordSimulator {
-    public static String getPassword(){
+    public static String getPassword() {
         List<String> predefinedPasswords = new ArrayList<>();
         predefinedPasswords.add("vejaS05");
         predefinedPasswords.add("zaiBas10");
```

Comment:
As far as I remember we agreed on a different algorithm for generating passwords. Since this algorithm can generate only 5 passwords so it's not secure.